src/components/KuadrantPoliciesPage.tsx

```diff
@@ -0,0 +1,422 @@
+import * as React from 'react';
+import { useParams } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+import { Dropdown, DropdownItem, DropdownList, MenuToggle, MenuToggleElement, Alert, AlertGroup, Title} from '@patternfly/react-core';
+import { sortable } from '@patternfly/react-table';
+import EllipsisVIcon from '@patternfly/react-icons/dist/esm/icons/ellipsis-v-icon';
+import { useK8sWatchResource, K8sResourceCommon, ResourceLink, useActiveNamespace, HorizontalNav, useListPageFilter,
+  ListPageBody,
+  ListPageCreate,
+  ListPageFilter,
+  VirtualizedTable,
+  TableData,
+  RowProps,
+  TableColumn,
+  NamespaceBar,
+  Timestamp,
+  useActivePerspective,
+ } from '@openshift-console/dynamic-plugin-sdk';
+import './kuadrant.css';
+
+interface Resource {
+  name: string;
+  gvk: {
+    group: string;
+    version: string;
+    kind: string;
+  };
+}
+
+interface ExtendedK8sResourceCommon extends K8sResourceCommon {
+  status?: {
+    addresses?: { value: string }[];
+  };
+}
+
+const statusConditionsAsString = (obj: any) => {
+  if (!obj.status || !obj.status.conditions) {
+    return '';
+  }
+  return obj.status.conditions
+    .map(condition => `${condition.type}=${condition.status}`)
+    .join(',');
+}
+
+const resources: Resource[] = [
+  { name: 'AuthPolicies', gvk: { group: 'kuadrant.io', version: 'v1beta2', kind: 'AuthPolicy' } },
```

Comment:
wondering to myself how we should generally approach our CRD versioning and our UI. I guess the UI will support certain versions, or perhaps be in lock step. wondering what we do with resources at different versions and the UI. expect we won't have too many differences in terms of what we show in tables and the like, but worth thinking about.

Reply:
Once we go v1, this will definitely be something to consider as soon as a new crd version is available.

I suspect the plugin code will keep in lockstep with the latest version (as long as that's what the 'stored' version is), and a conversion webhook would be required with the newer release of the operator to migrate older versions of crds to newer.
Both old & new versions would need to be supported for some time.